internal/policygen/load.go

```diff
@@ -49,7 +67,7 @@ func loadConfig(path string) (*config, error) {
 		return nil, fmt.Errorf("convert config to JSON: %v", err)
 	}
 
-	sj, err := hcl.ToJSON([]byte(schema))
+	sj, err := hcl.ToJSON(schema)
```

Comment:
Where is the `sj` variable used? I can't see it in this function after it's assigned.

Reply:
It's used at line 75